Jenkinsfile

```diff
@@ -50,159 +50,114 @@ pipeline {
             // TODO DDF-2971 refactor this stage from scripted syntax to declarative syntax to match the rest of the stages - https://issues.jenkins-ci.org/browse/JENKINS-41334
             steps {
                 parallel(
-                        linux: {
-                            node('linux-large') {
-                                retry(3) {
-                                    checkout scm
-                                }
-                                timeout(time: 3, unit: 'HOURS') {
-                                    // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
-                                    withMaven(maven: 'Maven 3.3.9', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}', options: [artifactsPublisher(disabled: true), dependenciesFingerprintPublisher(disabled: true, includeScopeCompile: false, includeScopeProvided: false, includeScopeRuntime: false, includeSnapshotVersions: false)]) {
-                                        sh 'mvn install -pl !$DOCS -DskipStatic=true -DskipTests=true -T 1C'
-                                        sh 'mvn clean install -B -T 1C -pl !$ITESTS -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/$CHANGE_TARGET'
-                                        sh 'mvn install -B -pl $ITESTS -nsu'
-                                    }
-                                }
+                    linux: {
+                        node('linux-large') {
+                            retry(3) {
+                                checkout scm
                             }
-                        },
-                        windows: {
-                            node('proxmox-windows') {
-                                bat 'git config --system core.longpaths true'
-                                retry(3) {
-                                    checkout scm
+                            timeout(time: 3, unit: 'HOURS') {
+                                // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
```

Comment:
❓  I realize this is an older comment. Is there a maven issue to link to this? Just curious how we will know if we can upgrade or will it be more trial and error to see if it is no longer an issue?

Reply:
So far we have only have trial and error. The failures we were seeing were not linked with any reported issue that I know of.